src/azure-cli/azure/cli/command_modules/rdbms/_util.py

```diff
@@ -103,3 +103,20 @@ def call(*args, **kwargs):
                 sleep(interval_sec)
         return call
     return decorate
+
+
+def get_index_tuning_settings_map():
+    return {
```

Comment:
I gave this translation a second thought, and have decided that the mapping should only remove the `index_tuning.` prefix, but it shouldn't translate `_` <=> `-`.

That idea only made sense when we were considering that we would expose those names as names of parameters for the CLI command. But, since we've model it so that these are values passed as values to the `--name` parameter, it makes sense to not translate between hyphens and underscores.

Reply:
Will change to them having to pass in with '_' as shown from configuration name

Reply:
```
az postgres flexible-server index-tuning show-settings: 'failsetting' is not a valid value for '--name'. Allowed values: analysis_interval, max_columns_per_index, max_index_count, max_indexes_per_table, max_queries_per_database, max_regression_factor, max_total_size_factor, min_improvement_factor, mode, unused_dml_per_table, unused_min_period, unused_reads_per_table.

az postgres flexible-server index-tuning show-settings --ids /subscriptions/ac0271d6-426b-4b0d-b88d-0d0e4bd693ae/resourceGroups/nasc-runner/providers/Microsoft.DBforPostgreSQL/flexibleServers/nasc-indextuning --name unused_reads_per_table
{
...
  "name": "index_tuning.unused_reads_per_table",
```